src/twisted/newsfragments/9488.feature

```diff
@@ -0,0 +1 @@
+Implement the AMP version 2 protocol to allow values greater than 64k.
```

Comment:
We shouldn't have duplicate NEWS entries like this, as it produces useless duplicative noise in the changelog.  The correct thing to do here is to close the second ticket as a duplicate in trac, then have there be one NEWS file.

Reply:
You might want to update the guidelines in this case, from the news fragment instructions: " You should replace <ticket number> with the ticket number which is being resolved by the change (if multiple tickets are resolved, multiple files with the same contents should be added). "

Reply:
@jothan Hmm. Apparently I'm wrong about this.  I thought that this was an idiom that predated our usage of towncrier, but apparently towncrier adopted this same idiom of hashing the contents of the files; I just created a sample project to test it out and it does the right thing in all the edge cases that I thought would not be handled.  Sorry for the mistake!  I am still probably going to be rolling this back out for the other reasons discussed below, but at least we can forget about this...